Setting up initial files

rommeltorquator opened the pull request:
## Pull Request
#### Building the basic functionalities

## The Story
I just started building the app Sunday and I only have 2 weeks to finish this. I believe that it will be possible to finish this app on time given that there would be no more interruption on my end. Here is what I did in the PR.

- Installed the necessary dependencies, gems, files
- Implemented the initial database schema
- Added home and procedure controller
- Added 2 user types
- Created initial html files
- Built the initial admin and member functionalities
- Implemented the search functionality

## Screenshots
#### Sign in page
![image](https://user-images.githubusercontent.com/35893311/125441389-b9e90eab-d136-4bbe-ae60-21658df04c59.png)
#### Sign up page
![image](https://user-images.githubusercontent.com/35893311/125441430-f56bdfc3-b575-4319-8b6f-d0f86800b735.png)
#### Member home page
![image](https://user-images.githubusercontent.com/35893311/125441476-62c19895-5e92-4f31-87af-f848314efe4e.png)
#### Member procedure index page and search field
![image](https://user-images.githubusercontent.com/35893311/125441535-ad72561f-4a87-4718-9951-09af4736b3fc.png)
#### Admin home page
![image](https://user-images.githubusercontent.com/35893311/125441694-349af9bf-f921-44a8-9372-d35dcb574004.png)
#### Admin procedure index page and search field
![image](https://user-images.githubusercontent.com/35893311/125441886-8c33703a-cfda-4105-bf90-a46cf0133c05.png)

rommeltorquator committed:
Working on the initial functionalities

rommeltorquator committed:
Working on the initial functionalities

rommeltorquator committed:
Added two user types

rommeltorquator committed:
Building the initial functionalities

rommeltorquator committed:
Building the initial functionalities

rommeltorquator committed:
Building the initial functionalities

analyn-cajocson commented on Gemfile:
```diff
@@ -37,3 +42,5 @@ group :test do
   gem 'rubocop-rspec', require: false
 end
+
+gem 'image_processing', '~> 1.12', '>= 1.12.1'
```
Can you please move this to the upper part of the `Gemfile`?

analyn-cajocson commented on app/controllers/procedures_controller.rb:
```diff
@@ -0,0 +1,64 @@
+    # @q = Procedure.ransack(params[:q])
+
+    @posts = @q.result(distinct: true)
+    # @posts = Procedure.where(team_id: current_member.team.id).order(id: :desc) if member_signed_in?
+    # @posts = Procedure.where(team_id: current_admin.team.id).order(id: :desc) if admin_signed_in?
```
remove bad comments

analyn-cajocson commented on Gemfile:
```diff
@@ -7,14 +7,19 @@ gem 'rails', '~> 6.0.3', '>= 6.0.3.4'
 
 gem 'devise'
-gem 'hamlit-rails'
+# gem 'hamlit-rails'
```
remove this comment

rommeltorquator committed:
Building the initial functionalities

rommeltorquator committed:
- Installed the necessary dependencies, gems, files
- Implemented the initial database schema
- Added home and procedure controller
- Added 2 user types
- Created initial html files
- Built the initial admin and member functionalities
- Implemented the search functionality
